Address Stanislav nits (#325)
cfrg/draft-irtf-cfrg-hash-to-curve: draft-irtf-cfrg-hash-to-curve.md

side-channel vulnerabilities. See Dragonblood {{VR20}} as one example of this
problem in practice.

> This document is a product of the Crypto Forum Research Group (CFRG) in the IRTF.

Reviewer: ```suggestion
This document represents the consensus of the Crypto Forum Research Group (CFRG).
```

I think Stanislav's email requested the above line at the end of the intro.
